CHANGELOG.adoc

```diff
@@ -371,6 +371,7 @@ In other scenarios, the default creation position border remains the eastern one
 - https://github.com/eclipse-sirius/sirius-web/issues/3220[#3220] [deck] Add documentation
 - https://github.com/eclipse-sirius/sirius-web/issues/3178[#3178] [portal] Add documentation for portals
 - https://github.com/eclipse-sirius/sirius-web/issues/2890[#2890] [portal] Display the contents of a Portal in the Representations view
+- https://github.com/eclipse-sirius/sirius-web/issues/2837[#2837] [gantt] Enhance gantt scroll bar management
```

Comment:
Please, move this line in the current version (v2024.5.0)

Reply:
done